frontend/packages/dev-console/src/components/topology/filters/FilterDropdown.tsx

```diff
@@ -0,0 +1,55 @@
+import * as React from 'react';
+import { Select, SelectVariant, SelectOption, SelectGroup } from '@patternfly/react-core';
+import { ShowFiltersKeyValue, ExpandFiltersKeyValue, DisplayFilters } from './filter-utils';
+
+type FilterDropdownProps = {
+  filters: DisplayFilters;
+  onChange: (filter: DisplayFilters) => void;
+};
+
+const FilterDropdown: React.FC<FilterDropdownProps> = ({ filters, onChange }) => {
+  const [isOpen, setIsOpen] = React.useState();
```

Comment:
do we need default value for `isOpen`

Reply:
requested changes are obsolete now, as i rebased the dependant PR